src/objective/rank_objective.hpp

```diff
@@ -244,14 +245,12 @@ class LambdarankNDCG : public RankingObjective {
         const double low_discount = DCGCalculator::GetDiscount(low_rank);
 
         // If there is no position bias map, just use ratio of 1.0
-        double position_bias_ratio = min_pos_bias_;
+        double position_bias_ratio = 1.0;
         if (!position_bias_lookup_.empty()) {
-            auto it = position_bias_lookup_.find(std::make_pair(high, low));
+            auto it = position_bias_lookup_.find(std::make_pair(std::min(high+1, position_bias_lookup_max_), std::min(low+1, position_bias_lookup_max_)));
```

Comment:
I added the +1 because I **_think_** low and high are 0-based whereas the csv we use is 1-based but definitely need a double check on this

Reply:
maybe add a comment to explain why high+1. just to make sure we are not confused the next time we look at it.